### ISSUE TITLE
libftdi1.2.dylib not loaded

### ISSUE THREAD

**Issue by davesims** [kubostech/kubos-sdk#64]
Ran into this trying to flash:

```bash
➜  kubotest kubos flash
/usr/local/lib/python2.7/site-packages/kubos/bin/osx/openocd -f 
/usr/local/lib/python2.7/site-package s/kubos/flash/openocd/stm32f407vg.cfg -s 
/usr/local/lib/python2.7/site-packages/kubos/flash/openocd - c 
"stm32f4_flash /Users/davesims/kubos/kubotest/build/stm32f407-disco-gcc/source/kubotest" 
dyld: Library not loaded: /usr/local/lib/libftdi1.2.dylib   
Referenced from: /usr/local/lib/python2.7/site-packages/kubos/bin/osx/openocd   
Reason: image not found /usr/local/lib/python2.7/site-packages/kubos/flash/openocd/flash.sh: line 29: 77092 Trace/BPT trap: 5        $openocd -f $this_dir/$cfg -s $search_path -c "$cmd"
```
/cc @kubostech/devs

**Comment by kyleparrott** [comment]
Thanks @davesims, Looking into it.

**Comment by kyleparrott** [comment]
This will be fixed in the next release. If you need an immediate fix, manually install openocd: 

`brew install openocd`

**Comment by davesims** [comment]
Awesome, thanks @kyleparrott!! :bow:

**Comment by kyleparrott** [comment]
This should have been fixed as part of #65
